pkgs/development/python-modules/pyxdg/default.nix

```diff
@@ -1,30 +1,24 @@
-{ stdenv
+{ lib
 , buildPythonPackage
-, fetchPypi
-, fetchpatch
+, fetchFromGitLab
+, nose
 }:
 
 buildPythonPackage rec {
   pname = "pyxdg";
-  version = "0.26";
+  version = "0.27";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "fe2928d3f532ed32b39c32a482b54136fe766d19936afc96c8f00645f9da1a06";
+  src =  fetchFromGitLab {
+    domain = "gitlab.freedesktop.org";
+    owner = "xdg";
+    repo = pname;
+    rev = "rel-${version}";
+    sha256 = "1dg826vrc7ifkk4lnf648h61cqfamaqmngkn9hgmxnf9gqmkbn0k";
   };
 
-  # error: invalid command 'test'
```

Comment:
Can you please add a comment here with something like ``# tests fail on NixOS``?

Reply:
Done